localization/AppStoreDescription.txt

```diff
@@ -1,3 +1,8 @@
+Title: Edge - Bitcoin, Ethereum, Monero, Ripple Wallet
+-----
+Short description: Private, secure & easy to use. Buy, sell, send & receive Bitcoin & other crypto.
```

Comment:
should we say cryptocurrencies here instead of crypto? Maybe take a look at ASO keywords and what would garner the best search results

Reply:
@jamesrisberg This is actually the title as chosen by our ASO company. They are actively experimenting with the wording.